### PR TITLE
Add typings for reach-ui utils

### PR DESCRIPTION
**Pull request by 102**

### CODE REVIEW

File: definitions/npm/@reach/visually-hidden_v0.1.x/flow_v0.84.x-/visually-hidden_v0.1.x.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// @flow
+
+declare module '@reach/visually-hidden' {
+  declare module.exports: React$StatelessFunctionalComponent<{
+    children: React$Node
```
Comment (102) [inline definitions/npm/@reach/visually-hidden_v0.1.x/flow_v0.84.x-/visually-hidden_v0.1.x.js]: https://ui.reach.tech/visually-hidden

https://github.com/reach/reach-ui/blob/master/packages/visually-hidden/src/index.js

File: definitions/npm/@reach/auto-id_v0.2.x/flow_v0.84.x-/auto-id_v0.2.x.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// @flow
+
+declare module '@reach/auto-id' {
```
Comment (102) [inline definitions/npm/@reach/auto-id_v0.2.x/flow_v0.84.x-/auto-id_v0.2.x.js]: https://github.com/reach/reach-ui/blob/master/packages/auto-id/src/index.js

File: definitions/npm/@reach/auto-id_v0.2.x/flow_v0.84.x-/auto-id_v0.2.x.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  declare module.exports: {
```
Comment (pascalduez) [inline definitions/npm/@reach/auto-id_v0.2.x/flow_v0.84.x-/auto-id_v0.2.x.js]: Prefer ES modules declarations, especially if the package use them.

File: definitions/npm/@reach/visually-hidden_v0.1.x/flow_v0.84.x-/visually-hidden_v0.1.x.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+// @flow
+
+declare module '@reach/visually-hidden' {
+  declare export default React$StatelessFunctionalComponent<{
+    children: React$Node
+  }>;
```
Comment (pascalduez) [inline definitions/npm/@reach/visually-hidden_v0.1.x/flow_v0.84.x-/visually-hidden_v0.1.x.js]: You must add `...` here, it's an inexact object, sorry I missed it earlier. 
Surprising the tests doesn't brake on it though.

Comment (102) [reply definitions/npm/@reach/visually-hidden_v0.1.x/flow_v0.84.x-/visually-hidden_v0.1.x.js]: Yikes, nice catch! I'm using `v0.84` with objects not being exact by default, will fix this soon

**Pull request merged**